src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
This class JpaRepository adds functionality that is not needed. It would be better just to use Repository<Book, Long> instead.

Reply:
Solved.